gnomad_constraint/resources/resource_utils.py

```diff
@@ -0,0 +1,37 @@
+from gnomad.resources.resource_utils import TableResource, VersionedTableResource
+
+# noqa: D100
+constraint_tmp_prefix = "gs://gnomad-tmp/constraint"
+# A context table annotated with VEP, coverage, and methylation information.
+annotated_context_ht = VersionedTableResource(
+    default_version="85",
+    versions={
+        "85": TableResource(
```

Comment:
I used the same version as `vep_context` table. Just double check this number make sense.